snmp/datadog_checks/snmp/data/auto_conf.yaml

```diff
@@ -73,3 +73,13 @@ instances:
     ## Name of your context (optional SNMP v3-only parameter).
     #
     context_name: "%%extra_context_name%%"
+
+    ## @param tags - list of key:value element - optional
+    ## List of tags to attach to every metric, event and service check emitted by this integration.
+    ##
+    ## Learn more about tagging: https://docs.datadoghq.com/tagging/
+    #
+    tags:
+      # The autodiscovery subnet the device is part of.
+      # Used by Agent autodiscovery to pass subnet name.
+      - "autodiscovery_subnet:%%autodiscovery_subnet%%"
```

Comment:
Any reason not to use the tag `network`?

Reply:
`autodiscovery_subnet` seems more explicit and avoid conflicting with:
https://github.com/DataDog/integrations-core/blob/585cca2b52d933aae954dd8a56b596df513682e7/snmp/datadog_checks/snmp/data/profiles/meraki-cloud-controller.yaml#L35-L38

Reply:
^That conflict looks like it should be fixed - right now it does conflict with the regular `network` tag used during autodiscovery… (Or our tag should be migrated to `autodiscovery_subnet`, I actually like this name)

Any case, should we have a "reserved tags" policy in place, similar to how the Agent forbids/discourages `hostname`, `device`, etc., would it be a good idea?

Reply:
> right now it does conflict with the regular network tag used during autodiscovery

Not really for now, since `network` tag is currently used for `snmp.discovered_devices_count` only https://github.com/DataDog/integrations-core/blob/1cac1dce94c93ae5b10aee49d70f565963e85e34/snmp/datadog_checks/snmp/snmp.py#L335-L337

That said, if we decide to `autodiscovery_subnet`, we should migrate by adding a new tag (not remove old one to keep backward compatibility).

> "reserved tags" policy in place

Yes, agree, good idea, in `README.md` for example.